provisiondomainmanager_policy.tf

```diff
@@ -198,10 +189,10 @@ data "aws_iam_policy_document" "provisiondomainmanager_policy_ecr_ecs_elb_events
     ]
 
     resources = [
-      "arn:aws:elasticloadbalancing::${local.userservices_account_id}:listener/*/domain-manager-*",
-      "arn:aws:elasticloadbalancing::${local.userservices_account_id}:listener-rule/*/domain-manager-*",
-      "arn:aws:elasticloadbalancing::${local.userservices_account_id}:loadbalancer/*/domain-manager-*",
-      "arn:aws:elasticloadbalancing::${local.userservices_account_id}:targetgroup/domain-manager-*",
+      "arn:aws:elasticloadbalancing:*:${local.userservices_account_id}:listener/*/domain-manager-*",
```

Comment:
Use tags to restrict these actions.

Reply:
@jsf9k Same question as above. Is there something wrong limiting access by resource?

Reply:
Somehow my comments are in the wrong places I think.